src/gui/servernotificationhandler.h

```diff
@@ -35,11 +35,8 @@ class ServerNotificationHandler : public QObject
 public slots:
     void slotFetchNotifications(AccountStatePtr ptr);
 
-private slots:
-    void slotNotificationsReceived(const QJsonDocument &json, int statusCode);
-
 private:
-    QPointer<JsonApiJob> _notificationJob;
+    void slotNotificationsReceived(JsonApiJob *job, const AccountStatePtr &accountState);
```

Comment:
I guess you should use `private Q_SLOTS` if this is a slot.

Reply:
I don't think we want to. I just thought it'd make sense code style wise. It shouldn't have any side effects, or does it e.g., increase compilation time?